app/components/items/item.tsx

```diff
@@ -0,0 +1,58 @@
+import ImageViewer from "./clover/image/image";
+import CloverImageViewer from "./clover/image/viewer";
+import AudioViewer from "./clover/audio/viewer";
+import VideoViewer from "./clover/video/viewer";
+import BookViewer from "./clover/book/viewer";
+import PDFViewer from "./clover/pdf/viewer";
+
+const Item = ({ item }) => {
+  let viewer;
+  switch (item.typeOfResource) {
+    case "still image":
+      if (item.isSingleCapture) {
+        viewer = (
+          <>
+            <h2> Image: {item.title} </h2>
+            <ImageViewer imageID={item.capture.imageID.$} />
+          </>
+        );
+      } else {
+        viewer = (
+          <>
+            <h2> Image: {item.title} </h2>
+            <CloverImageViewer />
+          </>
+        );
+      }
+      return viewer;
+    case "moving image":
+      viewer = (
+        <>
+          <h2> Video: {item.title} </h2>
+          <VideoViewer />
+        </>
+      );
+      return viewer;
+    case "sound recording":
+      viewer = (
+        <>
+          <h2> Audio: {item.title} </h2>
+          <AudioViewer />
+        </>
+      );
+      return viewer;
+    case "text":
+      //also PDF
+      viewer = (
+        <>
+          <h2> Book: {item.title} </h2>
+          <BookViewer />
+        </>
+      );
+      return viewer;
+    default:
+      return <h1>No type of resource match</h1>;
```

Comment:
Just an FYI: this is not a necessary update for this PR but in the future, there should only be one `h1` on the page and I don't think it should come from this component. I think that should be set at the page level so this component can just render the viewer or an error message without having to worry about the heading hierarchy of the page.